src/scripts/site.js

```diff
@@ -1129,13 +1129,13 @@ var AppInfo = {};
                 var itemTypes = ['Playlist', 'TvChannel', 'Program', 'BoxSet', 'MusicAlbum', 'MusicGenre', 'Person', 'Recording', 'MusicArtist'];
 
                 if (itemTypes.indexOf(itemType) >= 0) {
-                    return 'itemdetails.html?id=' + id + '&serverId=' + serverId;
+                    return 'details?id=' + id + '&serverId=' + serverId;
                 }
 
                 var contextSuffix = context ? '&context=' + context : '';
 
                 if ('Series' == itemType || 'Season' == itemType || 'Episode' == itemType) {
-                    return 'itemdetails.html?id=' + id + contextSuffix + '&serverId=' + serverId;
+                    return 'details?id=' + id + contextSuffix + '&serverId=' + serverId;
```

Comment:
Rather than use an alias, I would just rename the page `details` since that's accurate enough to make sense.

Reply:
The goal is to decouple the path from the name of the endpoint. Currently, the path is used for everything (Path to the HTML and name of the endpoint).

This gives more flexibility, allowing you to separate the two.

We won't have that issue when we move to `vue-router` since at that point it'll be endpoint + component, but in the meantime, I figured this would make for a nice addition.